src/Simplex/Chat/Messages.hs

```diff
@@ -505,13 +505,17 @@ rcvGroupEventToText = \case
   RGEMemberAdded _ p -> "added " <> profileToText p
   RGEMemberConnected -> "connected"
   RGEMemberLeft -> "left"
+  RGEMemberRole _ r -> "role " <> safeDecodeUtf8 (strEncode r)
```

Comment:
Include member in text representation, e.g. using `profileToText` - otherwise it's unclear whose role changed